### DISCUSSION TITLE
HD and other reading based mods score multiplier

### DISCUSSION THREAD

**Discussion by PMorghK** [ppy/osu#20476] [Polls]
The problem with scoring is that it is a very complex topic in this game, especially when mods are applied. 

This poll specifically targets reading mods such as HD(Hidden), TC(Tracable), AD(approach Different), and etc. The problem is that people can specifically use these mods instead of reading the game without mods, thereby playing better with these mods compared to no mod,which means that people should technically have score multipliers when there is no mods enabled.

Further more, in most senarios, mods like HR are clearly harder than HD  and this is why I suggest score multiplyers for reading based mods to be set to 1.03 instead of 1.06.

**Comment by Zaetherr** [comment]
osu < friday night funkin

**Comment by tematikkp** [comment]
nerf HD to 1.05x instead, buff HR to 1.07x

Because it depends on the map, like using HR in [Violet Perfume](https://osu.ppy.sh/beatmapsets/1010865) makes it drastically harder due to higher circle size difficulty. And using HD needs a little bit of memory.

**Comment by ghost** [comment]
See? This is why whitecat is making a fucking why HR weighting should be higher then HD. Keep HD the same but make HR x1.08

**Comment by Rekunan** [comment]
> people should technically have score multipliers when there is no mods enabled.

No, that's like saying a speed player should be awarded more pp for accurately streaming 180bpm compared to 270bpm because it's way below their comfort range, meaning they constantly overstream it by accident. Streaming 270bpm is objectively harder than streaming 180bpm, and reading mods such as HD make the game objectively harder compared to NM. Being able to shoot at a target only after waiting for it to become invisible is harder than being able to shoot at a target only after waiting a bit, and it's still fully visible.

> in most senarios, mods like HR are clearly harder than HD

What about the scenarios where HD is clearly harder than HR? Are you saying you're ok with the leaderboards of reading maps, like [Pudding Funfair](https://osu.ppy.sh/beatmapsets/993369#osu/2077273), to be even more dominated by HR?

The solution is to use PP scoring as it will be in a good position to be able to accurately compare plays against one another, especially once more work is done, such as having reading pp.

**Reply by PMorghK** [reply]
good example. minor problem. The map is also harder in nomod if compared to hr because of the low ar. I would argue that low ar is a completely different topic and should not even be considered in this discussion because some low ar maps are much easier to read compared to when they are read no mod and that is because of the fade that hd has.

basically what I am trying to say is that factoring in low ar is not a good example of hd being harder. its not. that is because most players are not at all good at reading low ar in general no matter hd or no mod. thereby hard rock is better for both the hd and nomod player in a low ar map. 

the solution also is not pp leaderboards because you are completely ignoring tournaments. also I win any way and hd should have a lower multiplier LMAO 
HD play PP:
![image](https://user-images.githubusercontent.com/43874888/195965779-7789c9aa-77dd-4d87-addb-99cdd6c90215.png)

HR play PP:
![image](https://user-images.githubusercontent.com/43874888/195965799-34317b79-da1c-47bd-adf0-b1a474a81021.png)

**Reply by Rekunan** [reply]
? You're gonna have to give me some examples of this, cause as a low AR player myself, I find that all low AR maps are harder to read HD than NM.

I agree, most players are bad at low AR/high density, but I don't see why that should matter. We should be looking at objective difficulty, and for low AR/reading maps, HD is more difficult than HR, and HR is more difficult than NM.

That's just cause the pp system isn't in a good state yet. There is a reading rework currently in development that will be able to correctly weigh these types of maps. You say that I'm ignoring tournaments, but once changes such as combo scaling and stat acc go in, I'd argue we can use pp as the win condition in tournaments, which I also believe is planned to be a future win condition option.

**Reply by PMorghK** [reply]
again I am arguing that if more people played low ar then the difference between low ar hd and low ar nomod would be less apparent than it currently is.

even if you want to look at it objectively here is some math. about 90 percent of maps are not considered low ar. that means the majority of maps are by your words not going to be reading heavy and thereby going to be comparatively easier to read with hd. thereby based on just objective math we should make HD multiplier lower than what it currently is. yea its not perfect but it is better than having the multiplier be exactly the same as hr

also if pp is such a great system it should be made better as soon as possible instead of changes being pushed every 6 months.
and again I also doubt the reading change will actually give more pp with hd because again, its not that hard of a mod once you learn how to play it.

**Reply by ghost** [reply]
feel free to correct me if i'm wrong, but what i'm understanding from this discussion is that score should be deprecated and pp should be used as a ranking system for map leaderboards instead. as a result, there would be no need for mod multipliers.

in that case i don't get why any of this is relevant. the discussion is about mod multipliers, after all. maybe ranking metrics (whether for player leaderboards, map leaderboards, tournaments, etc) could be discussed another time?

**Reply by PMorghK** [reply]
I really just want to make score more fair. I really don't care about what pp does yea. that's why I created this poll in the first place to see if the multiplier should be changed. I feel like most people would use score as the main metric when they start out and I want it to make more sense. thats why I have multiple posts here in regarding score calculations. and regarding tournaments, again I dont see why it should be replaced by pp because even people in the pp discord believe that pp is not at all perfect. by no means my system is better but if score v2 uses 1.06 for hd and 1.10 for hr and 1.20 for dt, I dont see why we cant do the same in lazer.

**Reply by smoogipoo** [reply]
Just to put an end to this conversation before it goes too far: score is not at all being deprecated. PP leaderboards are a separate entity.

**Comment by PMorghK** [comment]
if people think that pp is the answer, you are wrong. it just proves that hd multiplyer should be much lower than hr:

The map here I got from the comments which people tell me is considered reading heavy so its hd pp should be higher than usual.

But there is legit no point to HD if both mods are treated as equally challenging in score multiplier. Becuaee hr has the same multiplier but 50% more pp.

hr PP:
![image](https://user-images.githubusercontent.com/43874888/195965988-97700e8b-1b5c-4e7d-9ef8-f42989d39402.png)

hd PP:
![image](https://user-images.githubusercontent.com/43874888/195966002-951a1294-7875-4f63-94d7-54b86c48dc19.png)
